participants/matthias-stemmler.json

```diff
@@ -0,0 +1,52 @@
+// vim: ft=jsonc
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Matthias",
+		"familyName": "Stemmler",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "ms-tng",
+	// company name (optional)
+	"company": "TNG Technology Consulting GmbH",
+	// either both days or at least one day has to be set to true.
+	// If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+	"when": {
+		// June 27th, 2025
+		"friday": true,
+		// June 28th, 2025
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": true,
+	// your current interests (JS and in general) (required)
+	"tags": ["Kotlin", "Nx", "React", "Rust", "State management", "Tauri", "TypeScript", "Vite"],
+	// if you only eat vegan food (optional)
+	"vegan": true,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": true,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "I use it daily in my job as a Software Consultant and in private projects",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "Knowledge/experience in React, build tooling (e.g. Nx, Vite) and Rust",
+	// if you want a T-Shirt we need your size and variant preference (optional)
+	// the following sizes are available: S, M, L, XL, 2XL, 3XL (only regular cut)
+	"tShirtSize": "M",
+	// your Mastodon URL (optional)
+	"mastodon": null,
+	// your LinkedIn profile URL
+	"linkedin": "https://de.linkedin.com/in/matthias-stemmler-61607ab3",
+	// your X (Twitter) handle - must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$ (optional)
+	"X": "yourhandle",
```

Comment:
Do you want to remove this part, if you don't have an X account to share, please?